redditdownload/gfycat.py

```diff
@@ -23,17 +23,17 @@ def __init__(self):
         super(gfycat, self).__init__()
 
     def __fetch(self, url, param):
-        import urllib2
+        import urllib.request, urllib.error, urllib.parse
```

Comment:
other thing we can do is to remove it from our program and let user directly install from the source (or our forked source)

https://github.com/nim901/gfycat